src/components/Common/ReorderableList/ReorderableList.stories.tsx

```diff
@@ -0,0 +1,243 @@
+import type { Story } from '@ladle/react'
+import { useState } from 'react'
+import { I18nextProvider } from 'react-i18next'
+import { ReorderableList } from './ReorderableList'
+import { LocaleProvider } from '@/contexts/LocaleProvider/LocaleProvider'
+import { SDKI18next } from '@/contexts/GustoProvider/SDKI18next'
+
+// Centralized wrapper for all stories
+function GustoMockProvider({ children }: { children: React.ReactNode }) {
+  return (
+    <I18nextProvider i18n={SDKI18next}>
+      <LocaleProvider locale="en-US" currency="USD">
+        {children}
+      </LocaleProvider>
+    </I18nextProvider>
+  )
+}
```

Comment:
Nit: outside of scope but for a follow up we should potentially centralize the providers at a root place in ladle so these don't have to be redeclared at the story level